### ISSUE TITLE
bogus format string

### ISSUE THREAD

**Issue by Grazfather** [hugsy/gef#289]
This makes no sense.

https://github.com/hugsy/gef/blob/master/gef.py#L3543

What's the intent?
@d4em0n this is your code.

**Comment by d4em0n** [comment]
yes that is my code, it's should not be in there. i'm did not notice, sorry.

**Comment by Grazfather** [comment]
No problem, just wondering what the fix would be. Could you please file a PR?
